tools/src/launcher/Main.hs

```diff
@@ -515,11 +514,55 @@ spawnNode (path, args, mbLogPath) = do
             logInfo "Node has started"
             return (ph, asc)
 
-runWallet :: (FilePath, [Text]) -> M ExitCode
-runWallet (path, args) = do
+runWallet
+    :: Bool                               -- ^ wallet logging
+    -> (FilePath, [Text], Maybe FilePath) -- ^ Wallet, its args, wallet log file
+    -> (FilePath, [Text], Maybe FilePath) -- ^ Node, its args, node log file
```

Comment:
Don't pass redundant args, it's enough to pass `Maybe FilePath`, you ignore other two values.